### PR TITLE
Expose RequestOptions for Chat

### PR DESCRIPTION
**Pull request by Plootie**
As best as I can see there is currently no way to pass a `RequestOptions` instance to a `Chat` despite it supporting it. I've just added a public nullable property to the chat class to make it possible to pass it through again.

I'm unsure if this would be the preferable solution or if it's a better idea to add it as an overload/optional parameter. I can make those changes if you feel that's a better suit

### CODE REVIEW

**Comment by Plootie** [comment]
Oops. I forgot to rebase my fork ;P

**Comment by awaescher** [comment]
Would you rebase it please?

File: src/Chat.cs [hunk 1]
```diff
@@ -26,6 +27,8 @@ public class Chat
 	/// Gets or sets the AI model to chat with
 	/// </summary>
 	public string Model { get; set; }
+	
+	public RequestOptions? Options { get; set; }
```
Comment (awaescher) [inline src/Chat.cs:31]: If you add a summary to this property, I can merge it.

**Comment by Plootie** [comment]
Sorry about that. Forgot this PR existed for a few days. Should be all done now ;P

**Comment by awaescher** [comment]
Don't be sorry, you just made this a bit better. Just merged it, thanks a lot for your contribution!

**Pull request merged**